substrate/primitives/runtime/src/traits/mod.rs

```diff
@@ -1371,7 +1389,8 @@ pub trait ExtrinsicMetadata {
 	const VERSION: u8;
 
 	/// Signed extensions attached to this `Extrinsic`.
-	type SignedExtensions: SignedExtension;
+	// TODO: metadata-v16: rename to `Extension`.
+	type Extra;
```

Comment:
You're right. I renamed it to the original `SignedExtensions` just to have one less breaking change in this PR for whoever is relying on the `ExtrinsicMetadata` trait. It will change with the metadata update.

Reply:
I can see now.